godot-core/src/registry/property.rs

```diff
@@ -448,12 +454,34 @@ mod export_impls {
                 }
             }
         };
+
+        (@oneditor $Ty: ty) => {
+            impl GodotConvert for crate::obj::OnEditor<$Ty> {
+                type Via = <$Ty as GodotConvert>::Via;
+            }
+
+            impl Var for crate::obj::OnEditor<$Ty> {
+                fn get_property(&self) -> Self::Via {
+                    self.get_property().expect("Primitives can not be null.")
+                }
+
+                fn set_property(&mut self, value: Self::Via) {
+                    self.set_property(Some(value))
+                }
+            }
+
+            impl Export for crate::obj::OnEditor<$Ty> {
+                fn export_hint() -> PropertyHintInfo {
+                    <$Ty>::export_hint()
+                }
+            }
+        }
     }
 
     // Bounding Boxes
     impl_property_by_godot_convert!(Aabb);
-    impl_property_by_godot_convert!(Rect2);
-    impl_property_by_godot_convert!(Rect2i);
+    impl_property_by_godot_convert!(Rect2, oneditor);
+    impl_property_by_godot_convert!(Rect2i, oneditor);
```

Comment:
Also here, we should see if this is really necessary, or if a blanket impl is possible.

These individual trait `impl` are likely not great for compile times, and they also clutter the[ `OnEditor` doc page](https://godot-rust.github.io/docs/gdext/pr-1051/godot/obj/struct.OnEditor.html#method.init) massively:
![image](https://github.com/user-attachments/assets/dbe3cd00-9737-47f2-a0ea-32a12950716b)

Reply:
yeah, much better

![image](https://github.com/user-attachments/assets/a0268e73-f2c4-44da-8f7c-e832c06c46f7)